lib/safe-pg-migrations/configuration.rb

```diff
@@ -5,16 +5,17 @@
 module SafePgMigrations
   class Configuration
     attr_accessor(*%i[
+                    backfill_batch_size
+                    backfill_pause
                     blocking_activity_logger_margin
                     blocking_activity_logger_verbose
                     default_value_backfill_threshold
-                    backfill_batch_size
-                    backfill_pause
-                    retry_delay
                     max_tries
+                    retry_delay
                     sensitive_logger
+                    increase_lock_timeout_on_retry
```

Comment:
it seems you ordered all parameters there. Did you muss increase_lock_timeout_on_retry ?

Reply:
Add it after Mouad sorted, I'll fix that